src/tests/test_complex_claims.py

```
@@ -0,0 +1,358 @@
+        raise_exceptions=True,
+        include_debug_info=False,
+    )
+    return VerifactManager(config)
+
+
+@pytest.mark.asyncio
+@patch("src.verifact_manager.Runner.run")
+async def test_compound_claim(mock_run, manager):
+    """Test the pipeline with a compound claim that should be broken down."""
+    # A compound claim that contains multiple factual assertions
+    compound_claim = Claim(
```

Ruff (0.11.9)

Local variable `compound_claim` is assigned to but never used

Remove assignment to unused variable `compound_claim`

(F841)